src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -316,8 +316,8 @@ export default class AmalgamatingBusinesses extends Mixins(AmalgamationMixin, Co
   snackbar = false
   snackbarText = ''
   errorDialog = false
-  errorDialogText = undefined as string
-  errorDialogTitle = undefined as string
```

Comment:
This was actually causing some template errors. I originally had it this way so the default props would be used if we didn't specify them, but we always specify them anyway so '' is a better initial value.

Reply:
(From a high level, default prop values are normally used when you DON'T specify the prop at all. If we do specify the prop, but want to use its default value, then we'll have to set it to undefined. Neither of these applies here.)